Docs: Azure logs - cross reference field configuration display name (#26804)
grafana/grafana: docs/sources/features/datasources/azuremonitor.md

```kusto
Perf
| where $__timeFilter(TimeGenerated)
| summarize Samples=count(), ["Avg Value"]=avg(CounterValue)
    by bin(TimeGenerated, $__interval), Computer, CounterName, InstanceName
| order by TimeGenerated asc
```

> > **Tip**: In the above query, the Kusto syntax `Samples=count()` and `["Avg Value"]=...` is used to rename those columns — the second syntax allowing for the space. This will change the name of the metric that Grafana uses, and as a result, things like series legends and table columns will match what you specify. In this case, `Samples` will be displayed instead of `_count`.

Reviewer: ```suggestion
> **Tip**: In the above query, the Kusto syntax `Samples=count()` and `["Avg Value"]=...` is used to rename those columns — the second syntax allowing for the space. This changes the name of the metric that Grafana uses, and as a result, things like series legends and table columns will match what you specify. Here `Samples` is displayed instead of `_count`.
```

| order by TimeGenerated desc
```

### Format legend keys with aliases for Log Analytics

The default legend formatting is:

`metricName{dimensionName=dimensionValue,dimensionTwoName=DimensionTwoValue}`

> This can be customized by using the [display name field configuration option]({{< relref "../../panels/field-configuration-options.md#display-name" >}}).

Reviewer: ```suggestion
You can customize the legend formatting by setting the [display name field configuration option]({{< relref "../../panels/field-configuration-options.md#display-name" >}}).
```

> In order to change the metric name, you can change your query to specify a specific name. For example, `summarize avg(Active) by...` will return metrics with the name `avg_Active`. This can be changed to `Active` by changing that part of the query to be `summarize ["Active"]=avg(Active) by ...`.

Reviewer: Where are they changing the metric name? In the visualization? The legend? Both?

Also, instead of "this can be...", try writing "You can <do this>." It is both more active and often uses fewer words.

Author: The metric name is being changed in the data source query (a text field style query editor). So this will change the results returned from query, and therefore impact everything upstream.  I am not sure of a better way to say that than "your query"?

Author: So by SQL analogy, this would be similar to changing `Select avg(someColumn) FROM ...` to `SELECT avg(someColumn) AS Active FROM ...`

Reviewer: Try to write in present tense. "returns" rather than "will return", for example

Reviewer: For the query, I am trying to put myself in the user's shoes. As a user, why would I want to do this? Where would I see this change reflected?

Am I trying to change the variable name in the query so that my visualization is easier to understand, perhaps?

Author: This is more of a tip about the query language. So I have removed it. Also the title was wrong (not an alias) so hopefully this removes some of the confusion.

Reviewer: I think it's a great tip, just confusing where you had it. Consider adding a note or brief subsection somewhere else in this document, because I see this question A LOT in the community forums for various data sources.
